k6/run.sh

```
@@ -0,0 +1,260 @@
+echo "테스트 대상  : $BASE_URL"
+echo "시나리오     : $SCENARIO"
+echo "시작 시간    : $START_TIME"
+echo "=============================="
+
+case $SCENARIO in
+  smoke)
+    echo ""
+    echo "[Smoke Test 시작]"
+    echo "완료 후 k6/data/isbn_list.json을 준비해주세요."
+    echo ""
+    k6 run -e BASE_URL=$BASE_URL k6/scenarios/smoke.js
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)